platform/tron/api.go

```diff
@@ -40,14 +40,45 @@ func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
 	return txs, nil
 }
 
-/// Normalize converts a Tron transaction into the generic model
-func Normalize(srcTx *Tx) (tx blockatlas.Tx, ok bool) {
-	if len(srcTx.Data.Contracts) < 1 {
-		return tx, false
+func (p *Platform) GetTokenTxsByAddress(address, token string) (blockatlas.TxPage, error) {
+	tokenTxs, err := p.client.GetTxsOfAddress(address, token)
+	if err != nil {
+		return nil, err
+	}
+
+	var tokenInfo *Asset
+	if token != "" {
+		var wg sync.WaitGroup
+		wg.Add(1)
+			go func(id string) {
```

Comment:
Use `Gofmt` for automatically formats. You can use `cmd + option + L` to format the file inside Goland.